proto/lib/livebook_proto/agent_connected.pb.ex

```diff
@@ -18,4 +18,5 @@ defmodule LivebookProto.AgentConnected do
     json_name: "appDeployments"
 
   field :agents, 9, repeated: true, type: LivebookProto.Agent
+  field :org_active, 10, type: :bool, json_name: "orgActive"
```

Comment:
OK, we have a dependency issue here. We need to either:

1. Deploy Teams with this before we publish a new Livebook version
2. Rename field to `org_disabled : bool` or something like `org_status : active | disabled`

The reason is the default value for booleans is `false` so if Teams is not sending this value it is considered `false` which would block Teams features. The DSL allows a `default: true`, which would be my preference, but that is not available on proto3.

I guess option 1 is fine, option 2 with `:org_status` could be more future proof though. @jonatanklosko @josevalim @hugobarauna Thoughts?

Reply:
Making it an enum is probably going to be better in the long term?

Reply:
@josevalim https://github.com/livebook-dev/livebook/blob/04a93c0dccb68699db101b769766bfe5fc8ded17/proto/messages.proto#L164-L174

Reply:
We are going to deploy Teams first anyway, otherwise the new Livebook release would probably have issues, no?

Reply:
Yes, correct. We always need to deploy Livebook Teams first before we release a new Livebook.

Reply:
Yes, correct. We always need to deploy Livebook Teams first before we release a new Livebook. But if anyone uses main or nightly build, then it will be fully broken saying they need to pay. That's why we may also want to have an enum with better defaults? Or the enum would be broken as well?

Reply:
Enum works, but we cannot return more values in the future, which I assume you meant in "Making it an enum is probably going to be better in the long term?". We could make the status a string and make sure out-of-scope values don't break Livebook (though the question is what behaviour should these entail).

We could also use `optional`, which was reintroduced in proto3 and my understanding is that it decodes fields as `nil`, when not set. But that adds more complexity to the consumer.

So it may indeed be most straightforward to flip the flag to `inactive`, `disabled`, or similar.

Reply:
Why couldn't we return more values in the future? Is that Protobuf limitation or it's depend on us how we treat incoming values (e.g. exhaustive case/2 on exact values)?

Reply:
So let's ship with the inverse for now: `inactive`

Reply:
> Why couldn't we return more values in the future? Is that Protobuf limitation or it's depend on us how we treat incoming values (e.g. exhaustive case/2 on exact values)?

The protobuf library would keep an unknown value, but it would be an integer (see commit message in https://github.com/elixir-protobuf/protobuf/commit/6b8f40f206375c36666f2cc9a570a061f9f0d89a). Leaking the integer to the rest of the application is probably a bad idea, so we would need to account for the unknown value early. In some cases this could be as simple as picking a default value, but here I'm not sure if that's clear. Using strings has the benefit that it's safer to keep them around, but it mostly pushes the catch all and default behaviour problem to other parts of the app.

I don't remember if there was a specific scenario behind the `AppDeploymentStatusType` comment though.

Reply:
Im gonna go with inactive bool flag then. Though if anyone prefers can call it `disabled`, lmk. Thanks everyone!

Reply:
Sounds good to me!